src/Actions/CommandLine/Commands/VerticalNewFile.ts

```diff
@@ -0,0 +1,17 @@
+import * as vscode from 'vscode';
+import { CommandBase } from './Base';
+import VisualSplit from './VisualSplit';
+import NewFile from './NewFile';
+
+class VerticalNewFileCommand extends CommandBase {
+  constructor() {
+    super();
+  }
+  async execute(): Promise<void> {
+    await VisualSplit.execute();
+    await NewFile.execute();
+    await vscode.commands.executeCommand('workbench.action.closeOtherEditors');
```

Comment:
This may have unwanted effects when we execute the command in the third column.
We can use `workbench.action.files.newUntitledFile` then `workbench.action.moveEditorToNextGroup`.